docs/howto_config.md

```diff
@@ -10,48 +10,55 @@ The MicroShift configuration file must be located at `~/.microshift/config.yaml`
 The format of the `config.yaml` configuration file is as follows.
 
 ```yaml
-cluster:
-  clusterCIDR: ""
-  serviceCIDR: ""
-  serviceNodePortRange: ""
-  domain: ""
-  url: ""
-nodeIP: ""
-nodeName: ""
 subjectAltNames:
   - ""
-logVLevel: ""
+nodeName: ""
+nodeIP: ""
+url: ""
+network:
+  clusterNetwork:
+    - cidr: ""
+  serviceNetwork:
+    - ""
+  serviceNodePortRange: ""
+dns:
+  baseDomain: ""
```

Comment:
```suggestion
  baseDomain: ""
  clusterDomain: ""
```

baseDomain is used for public DNS/router records, while clusterDomain is for the service network domains, what typically is cluster.local